src/main/java/uk/gov/ons/census/fwmt/outcomeservice/message/OutcomeProcessPreprocessingDLQ.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.ons.census.fwmt.outcomeservice.message;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.amqp.core.Message;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.amqp.support.converter.Jackson2JsonMessageConverter;
+import org.springframework.amqp.support.converter.MessageConverter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.ons.census.fwmt.common.error.GatewayException;
+import uk.gov.ons.census.fwmt.outcomeservice.config.OutcomePreprocessingQueueConfig;
+
+import java.io.IOException;
+
+import static uk.gov.ons.census.fwmt.outcomeservice.config.OutcomePreprocessingQueueConfig.OUTCOME_PREPROCESSING_DLQ;
+
+@Slf4j
+@Component
+public class OutcomeProcessPreprocessingDLQ {
+
+  @Autowired
+  private ObjectMapper jsonObjectMapper;
+
+  @Autowired
+  private RabbitTemplate rabbitTemplate;
+
+  @RabbitListener(id = "rabbitDLQ", queues = OUTCOME_PREPROCESSING_DLQ, autoStartup = "false")
+  public void receiveMessage(Message message) throws GatewayException {
```

Comment:
could this infinitely move stuff from dlq to q?

Reply:
As discussed.